src/Evolution/DgSubcell/ReconstructionMethod.hpp

```diff
@@ -0,0 +1,40 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <iosfwd>
+
+#include "Options/Options.hpp"
+
+namespace evolution::dg::subcell::fd {
+/*!
+ * \brief The reconstruction method to use
+ */
+enum class ReconstructionMethod {
+  /// Dimension-by-dimension reconstruction assuming a tensor-product basis
+  DimByDim,
+  /// Reconstruct all dimensions at once requiring the integral over the
+  /// subcells and over the DG element to be equal.
```

Comment:
Remove everything starting from "requiring".  It's not a distinguishing property of this case.

Also make the corresponding change to the commit message for the next commit.

Reply:
Done and rebased! Thanks :)